workflow-templates/manage-labels.svg

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
According to the GitHub workflow template documentation, adding an icon is required. I can't comprehend why they would impose such a silly requirement, and my experiment showed that it will work without it, but I guess it's safest to just follow the rules.